client_seal.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+import "consensus_messages.proto"
+
+message ClientSealRequest {
+  bytes block_id = 1;
+}
+
+message ClientSealResponse {
+  enum Status {
```

Comment:
Please document what those entries should represent. I am not sure if we should use `STATUS_UNSET` (normally it don't occur).